independent-publisher-connectors/LINE WORKS Incoming Webhook/apiDefinition.swagger.json

```diff
@@ -0,0 +1,76 @@
+swagger: '2.0'

```

Comment:
The swagger file is not followed the indentation. 

![image](https://github.com/user-attachments/assets/b6c89d8a-7041-461f-be0a-55362f2d8c04)

Reply:
I have reviewed the indentation in the Swagger file and confirmed that it is consistent. The file is uniformly indented using 2 spaces. If there are specific lines or sections where you’ve noticed issues, could you please point them out? I’d be happy to make further adjustments based on additional guidance.

Reply:
Thank you, please remove the text highlighted in dark red color.